apps/site/components/Icons/Logos/Nodejs.tsx

```diff
@@ -11,173 +12,177 @@ const Nodejs: FC<NodeJsLogoProps> = ({
   className,
   variant = 'default',
   ...props
-}) => (
-  <svg
-    width="267"
-    height="80"
-    viewBox="0 0 267 80"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-    className={classNames('fill-[#333333] dark:fill-white', className)}
-    {...props}
-  >
-    <mask
-      id="mask0_1803_3458"
-      style={{ maskType: 'luminance' }}
-      maskUnits="userSpaceOnUse"
-      x="0"
-      y="0"
+}) => {
+  const t = useTranslations();
```

Comment:
@canerakdas should we use `getTranslations` since this component could be a server component?